packages/entities/src/lib/add.mutation.ts

```diff
@@ -36,7 +37,7 @@ export function addEntities<
   entities: OrArray<getEntityType<S, Ref>>,
   options: AddEntitiesOptions & BaseEntityOptions<Ref> = {}
 ): Reducer<S> {
-  return function (state, context) {
+  return function (state, context, action) {
```

Comment:
I prefer to put the action on the context and name it `actions`

Reply:
To understand when you say on the context: 

You need before:

```ts
return function (state, actions, context) {
```

or put the actions inside the context

```ts
export type ReducerContext = { config: Record<PropertyKey, any>, actions: EntityAction$ };
```

Reply:
export type ReducerContext = { config: Record<PropertyKey, any>, actions: EntityAction$ };